### DISCUSSION TITLE
Some questions about adverserial graph autoencoder

### DISCUSSION THREAD

**Discussion by HelloWorldLTY** [pyg-team/pytorch_geometric#6195] [Ideas]
Hi, I have a question about AGAE/AVGAE about its priniciple. Since the github page of this model is not being maintained for a long while, so I come to this fantastic community for help.

I am confusing about the intuitive point in this project. I think it is hard to determine the true prior distribution (like they used normal here), and our data may not have normal prior distribution. Therefore, I wonder what is the reason they used the adverserial training here. Thanks a lot.
(https://arxiv.org/pdf/1802.04407.pdf)

**Comment by rusty1s** [comment]
This is a great point, the use of Gaussian is a bit questionable here. There is a similar problem in VGAE which is further explained here: https://github.com/tkipf/gae/issues/20#issuecomment-446260981
